sros2.repos

```diff
@@ -0,0 +1,161 @@
+repositories:
```

Comment:
Actually the [update from yesterday](https://github.com/ros2/sros2/pull/3/commits/3a4d7dbe7e9c893aff97523492f546b9d11ace9a) should allow to remove this file completely. Looks like I forgot to commit the deletion and updated only the Dockerfile but not the other sections of the README. Thanks for pointing it out !